### DISCUSSION TITLE
How to display the preview in a NextJS app

### DISCUSSION THREAD

**Discussion by loupzeur** [remotion-dev/remotion#2263] [Q&A] [answered]
Hello,

I'd like to have the preview console (the one we have using the command : remotion preview ...) in a NextJS app to create a videos using APIs and Database, but so far, I only have managed to use the <Player /> to display the video.

Is there a components to display the preview with the timeline and others informations ?

Thanks

**Comment by JonnyBurger** [comment]
You can start the preview also in the Next.js app if you follow these steps: 

https://www.remotion.dev/docs/brownfield

**Reply by loupzeur** [reply]
in this case I only have the player, not the timeline and the details that the command display like this one :
![image](https://github.com/remotion-dev/remotion/assets/20752997/9e3b3699-0a42-4f1a-bf60-8aad30bb6e9a)

unless its a parameter to use in the Player ?

**Comment by JonnyBurger** [comment, marked as answer]
I see - no this timeline component you cannot use in a Next.js app.
There would be too many parametrization options required to make the style fit everyone.

So the way forward is to build your own timeline around the Player. We'll plan on making a component for this in the future!
